README.md

```diff
@@ -78,33 +83,34 @@ All content on this feed SHOULD be encrypted with box2 encryption.
 
 ### Group feeds
 
-Each group feeds MUST be a direct subfeed of a shard feed, where the shard is derived using the `groupKey` encoded in BFE. A group feed MUST be declared on the shard feed with `feedpurpose` equal to the `groupKey` encoded as an SSB URI string.
+Each group feeds MUST be a direct subfeed of a shard feed, where the shard is derived using the base64 encoded group `secret`.
+A group feed MUST be declared on the shard feed with `feedpurpose` equal to the same base64 encoded group `secret`.
```

Comment:
Is the group feed declared publicly or encrypted? If it's always an encrypted msg on the shard feed, then we could say that in this sentence. But if can be public, aren't we then leaking the sensitive `secret`?